src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+gremlin.endpoint=endpoint.gremlin.cosmosdb.azure.com
+gremlin.port=443
+gremlin.username=/dbs/database/colls/collection
```

Comment:
This username is a collection?

Reply:
yes, username consist of /dbs/${database}/colls/${collection} from gremlin side. Actually the string identified unique azure cosmosdb collection for all db APIs.